### ISSUE TITLE
Error when update database, try to delete crm stage in model data

### ISSUE THREAD

**Issue by ertus-odoo** [odoo/odoo#30328]
Impacted versions:
Odoo 12.0 Enterprise Edition, French database, Paris,French Timezone

Steps to reproduce:
Existing database with custom stage et some leads. Update database (only the module CRM) with last version of Odoo 12.

Current behavior:
During the update, he tries to delete CRM stages, but these stages are link to existing leads
In ir_model_data table, i have 7 lines for crm.stage model (stage_lead1, stage_lead2...stage_lead8). For exemple, stage_lead8 is link to res_id 22 in ir_model_data. CRM stage ID 22 exists and he has some leads. In all the projects, i didn't find data file with external ids greater than stage_lead4 which is in CRM module...

Expected behavior:
Just update module, not try to delete things...

Video/Screenshot link (optional):
https://drive.google.com/file/d/15xmGhrBGMuMAOQ8bcrprk8Vy9Wk7coTT/view?usp=sharing

**Comment by ertus-odoo** [comment]
Ok but there are no data in all code and the syntaxe looks like native code. Is it possible these datas where introduce in an old commit ? 

Thank you

**Comment by pedrobaeza** [comment]
Following the history of https://github.com/odoo/odoo/commits/12.0/addons/crm/data/crm_data.xml and with gitk for the string `stage8`, I'm not finding anything.

**Comment by ertus-odoo** [comment]
Ok I found it, it come from OpenERP. Does it mean when I import old datas, he will try to delete every datas which not exists anymore ? To solve the problem I have to delete ir_model_data manually in the database.

**Comment by pedrobaeza** [comment]
Yes, if the XML-ID belongs to an updated module that is not included in code.

**Comment by ertus-odoo** [comment]
> Yes, if the XML-ID belongs to an updated module that is not included in code.

Ok, lots of work is waiting for me! Thanks
